codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```diff
@@ -264,7 +288,7 @@ let updateRegister = (
     isFetchingAtHead,
     latestFetchedBlock,
     firstEventBlockNumber,
-    fetchedEventQueue: Array.concat(self.fetchedEventQueue, newFetchedEvents),
+    fetchedEventQueue: Array.concat(newFetchedEvents->Array.reverse, self.fetchedEventQueue),
```

Comment:
Letting you know that `Array.reverse` mutates the original array. I don't know whether it's a problem here

Reply:
Interesting, this is Belt.Array.reverse since it's opened at the top, is that the same?

Reply:
Checked the source code. Belt is fine, it's immutable

Reply:
Confirming I've tested and it does not mutate 👍🏼